hc/api/tests/test_list_checks.py

```diff
@@ -30,19 +31,23 @@ def setUp(self):
 
     def get(self):
         return self.client.get("/api/v1/checks/", HTTP_X_API_KEY="abc")
-
+    
     def test_it_works(self):
```

Comment:
This isn't a good name. Change it to the name that clarify the intent of the funciton